randomizer.py

```diff
@@ -80,6 +80,7 @@ def __init__(self, name):
         self._projects = None
         self._potionCards = None
         self._ways = None
+        self._actionCards = None
```

Comment:
The other variables do not end with "cards" (with the exception of allies and traits, which I think is a mistake). The only one that ends with "cards" is the `_potionCards`, which is because the cards themselves are not potions, they are cards with potions in their cost. This should be called `_actions`.

Reply:
Agreed.